### DISCUSSION TITLE
#discussioncomment-NNNNNNN links are sometimes broken

### DISCUSSION THREAD

**Discussion by willfaught** [community/community#49717] [Discussions]
### Select Topic Area

Product Feedback

### Body

Sometimes when I click on a link in a GitHub email containing a new comment to see that comment on the site, it doesn't work. It just scrolls to some random place, and I can't find the comment manually by expanding all the collapsed comments and searching for text from the email. And no, the comment wasn't deleted or changed.

**Comment by ghost** [comment]
Hi there! Sorry to hear that you're having trouble with broken links when trying to view comments on GitHub. This can be frustrating, but there are a few things you can try to fix the issue.

Firstly, try clearing your browser cache and cookies. Sometimes this can help resolve issues with links not working properly. You can find instructions for clearing your cache and cookies in your browser's settings.

If that doesn't work, you can try opening the link in a different browser or device to see if it works there. If it does, then the issue may be specific to the browser or device you were previously using.

Lastly, if neither of these solutions work, you can report the issue to GitHub support by opening a new support ticket. They may be able to help you troubleshoot the issue further and find a solution.

I hope this helps you fix the broken link issue and view the comments you're looking for! Let me know if you have any further questions or if there's anything else I can do to help.

**Reply by willfaught** [reply]
How do I open a new support ticket?

**Reply by ghost** [reply]
Sure, I'd be happy to help you with opening a new support ticket on GitHub.

To open a new support ticket, you can visit the GitHub Support page (https://support.github.com/contact) and select "Get help with GitHub features and products" as the reason for your inquiry. Then, fill out the form with a detailed description of the issue you're experiencing and any relevant information, such as the broken link and the email it came from. You can also attach any screenshots or error messages you've encountered to help provide more context.

Once you've submitted the form, the GitHub support team will review your inquiry and get back to you as soon as possible with a solution or further troubleshooting steps.

I hope this helps! Let me know if you have any further questions or concerns.

**Reply by willfaught** [reply]
Thanks!

**Reply by ghost** [reply]
> Thanks!

You're welcome! Let me know if you have any other questions or need further assistance.

**Comment by LindseyB** [comment]
This is a bug. Thanks for the report.
